graphql-dgs-client/src/test/kotlin/com/netflix/graphql/dgs/client/GraphqlSSESubscriptionGraphQLClientTest.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2023 Netflix, Inc.
+ * Copyright 2021 Netflix, Inc.
```

Comment:
More curious than an actual comment, but why did the (c) get adjusted back in time?  Should this be a SED to bump them up to 2024?

Reply:
Weird, I have no idea. The copyright header comes from a profile in Intellij, which has `$today.year` as a variable. After this is merged with can do a clean up.